conf/nginx/templates/nginx.conf.web.carbonio.admin.default.template

```diff
@@ -22,7 +22,13 @@ server
 
     location /
     {
-     return 404;
+        if ($http_cookie !~ "ZM_ADMIN_AUTH_TOKEN=") {
+            return 307 "/static/login/";
+        }
+
+        if ($http_cookie ~ "ZM_ADMIN_AUTH_TOKEN=") {
+            return 307 "/carbonioAdmin";
+        }
```

Comment:
LGTM, but I have a question: will this work with just one if? Example:
```suggestion
        if ($http_cookie !~ "ZM_ADMIN_AUTH_TOKEN=") {
            return 307 "/static/login/";
        }
        return 307 "/carbonioAdmin";
```
or vice-versa. Should be equivalent, just maybe less to type and a bit more straightforward (at least in my mind)? -> (If not cookie then login, else always redirect to /carbonioAdmin).
Just a suggestion, but either work for me :slightly_smiling_face:

Reply:
You are right, it will work.